aspnetcore/tutorials/web-api-help-pages-using-swagger.md

```diff
@@ -110,239 +80,217 @@ In Visual Studio, press ^F5 to launch the app and navigate to `http://localhost:
+The stock UI is both functional and presentable; however, when building documentation pages for your API, you want it to represent your brand or theme. Accomplishing that task with the Swashbuckle components requires adding the resources to serve static files and then building the folder structure to host those files.
 
-Add the `"Microsoft.AspNetCore.StaticFiles": "1.0.0-*"` NuGet package to the project.
+If targeting .NET Framework, add the `Microsoft.AspNetCore.StaticFiles` NuGet package to the project:
 
-Enable static files middleware.
-
-<!-- literal_block {"ids": [], "linenos": false, "xml:space": "preserve", "language": "csharp", "highlight_args": {"hl_lines": [4]}} -->
-
-```csharp
-// This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-   public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
-   {
-       // Enable static files middleware.
-       app.UseStaticFiles();
-
-       app.UseMvcWithDefaultRoute();
-
-        // Enable middleware to serve generated Swagger as a JSON endpoint.
-        app.UseSwagger();
-
-        // Enable middleware to serve swagger-ui (HTML, JS, CSS etc.), specifying the Swagger JSON endpoint.
-        app.UseSwaggerUI(c =>
-        {
-            c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
-        });
-   }
-   ```
+```xml
+<PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.0" />
+```
 
-Acquire the core *index.html* file used for the Swagger UI page from the `Github repository <https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/1.0.0-rc1/test/WebSites/CustomIndexHtml/wwwroot/swagger/index.html>`_ and put that in the `wwwroot/swagger` folder and also create a new `custom.css` file in the same folder.
+Enable the static files middleware:
 
-![Solution Explorer showing Swagger UI custom css and html files within the wwwroot folder of the application](web-api-help-pages-using-swagger/_static/custom-files-folder-view.png)
+[!code-csharp[Main](../tutorials/web-api-help-pages-using-swagger/sample/TodoApi/Startup.cs?name=snippet_Configure&highlight=3)]
 
-Reference *custom.css* in the *index.html* file.
+Acquire the contents of the *dist* folder from the [Swagger UI GitHub repository](https://github.com/swagger-api/swagger-ui/tree/2.x/dist). This folder contains the necessary assets for the Swagger UI page. Copy the contents of that folder into your *wwwroot/swagger/ui* folder:
 
-```html
-<link href='custom.css' media='screen' rel='stylesheet' type='text/css' />
-```
+![Solution Explorer showing Swagger UI custom CSS and HTML files in wwwroot](web-api-help-pages-using-swagger/_static/custom-files-folder-view.png)
 
-The following CSS provides a simple sample of a custom header title to the page.
+Create a *custom.css* file in the *wwwroot/swagger/ui/css* folder. The file should contain the following CSS, which customizes the page header:
```

Comment:
You should try  to avoid should. HOw about something like

Create a *wwwroot/swagger/ui/css/custom.css* file with the following CSS to customize the page header: